File: Program43.c
#include<stdio.h>
#include<stdlib.h>
#include<stdbool.h>

bool CheckOccurance(int Arr[],int iSize,int iNo){
    int iCount=0;
    int iCnt=0;
    bool bFlag=false;

    for(iCnt=0;iCnt<iSize;iCnt++){
        if(Arr[iCnt]==iNo){
            bFlag=true;
        }
    }

    return bFlag;
}
int main(){

    int *Arr=NULL;
    int iCnt=0,iLength=0, iValue=0;
    bool bRet=false;

    printf("enter the size of array:\n");
    scanf("%d",&iLength);


    Arr=(int *)malloc(iLength * sizeof(int));

    printf("enter elements of array:\n");
    for(iCnt=0;iCnt<iLength;iCnt++){
        scanf("%d",&Arr[iCnt]);

    }

    printf("enter the number you want to find :\n");
    scanf("%d",&iValue);

    bRet=CheckOccurance(Arr,iLength,iValue);

    if(bRet==true){
        printf("the element is present\n");
    }else
    {
        printf("the element is not present\n");
        /* code */
    }
    
    
    return 0;
}

File: Program43.java
import java.util.Scanner;

class Numbers{
    public int iValue=0;
 
    Numbers(int iNo){
        this.iValue=iNo;
    }

    public void DisplayFactors(){

        for(int iCnt=1;iCnt<iValue;iCnt++){
            if(iValue % iCnt == 0){
                System.out.println(iCnt);
            }
        }

        
    }
}
public class Program43 {
    public static void main(String arr[]){

        Scanner sobj=new Scanner(System.in);
        System.out.println("enter number");
        int iValue=sobj.nextInt();

        Numbers nobj=new Numbers(iValue);
        nobj.DisplayFactors();


    }
}
